docs/interfaces.md

```diff
@@ -379,3 +379,9 @@ Can manage timeservers directly separate from ``config core``.
 Can access the tpm device /dev/tpm0.
 
 * Auto-Connect: no
+
+### fwupd
+
+Allow fwupd to update UEFI capsule format firmware.
```

Comment:
Ehh, I see this is very specific to fwupd, so 'fwupd' is the proper name of the interface. Can you adjust interfaces.md to have: "Can access snaps providing the fwupd interface which gives privileged access to update UEFI capsule format firmware."